### PR TITLE
Take locale into account when ordering apps and ignore diacritics during search

### PR DESCRIPTION
**Pull request by Heychsea**
This PR slightly changes the way the app drawer sorts apps, taking into account the current locale. It also ignore diacritics when searching to help find apps faster.

### CODE REVIEW

**Comment by Heychsea** [comment]
Any plans to merge it someday? It would be quite useful for a number of people, including myself.

**Comment by tanujnotes** [comment]
Hey, sorry for the delay. I'll take a look this weekend and see what I can do.

File: app/src/main/java/app/olauncher/ui/HomeFragment.kt [hunk 1]
```diff
@@ -248,7 +248,7 @@ class HomeFragment : Fragment(), View.OnClickListener, View.OnLongClickListener
 
     private fun launchApp(appName: String, packageName: String, userString: String) {
         viewModel.selectedApp(
-            AppModel(appName, packageName, getUserHandleFromString(requireContext(), userString)),
+            AppModel(appName, null, packageName, getUserHandleFromString(requireContext(), userString)),
```
Comment (tanujnotes) [inline app/src/main/java/app/olauncher/ui/HomeFragment.kt:251]: packageName is null here. Wouldn't that cause a problem with launching apps?

Comment (Heychsea) [reply app/src/main/java/app/olauncher/ui/HomeFragment.kt:251]: The second parameter has been replaced by an optional Collationkey, the package name is the third parameter now.

**Pull request merged**